ddtrace/internal/processor.py

```diff
@@ -12,11 +13,71 @@
 
 
 @attr.s
-class TraceProcessor(object):
+class Stitcher(object):
+    _partial_flush_enabled = attr.ib(type=bool)  # type: bool
+    _partial_flush_min_spans = attr.ib(type=int)  # type: int
+    _traces = attr.ib(factory=dict, init=False)  # type: List[_Trace]
```

Comment:
Type seems wrong and you can actually pass the type as a kwarg `type=` with attr.